fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1041,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * Ensure that the objects cannot be pairtree child resources
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+
+        //  Following the approach undertaken for FedoraExportIT#shouldRoundTripOnePairtree
+        final String objName = response.getFirstHeader("Location").getValue();
+        final String pairtreeName = objName.substring(serverAddress.length(), objName.lastIndexOf('/'));
```

Comment:
That's because it isn't there. You're doing the best that can be done, so don't worry about it. This PR is looking really good now.

Just on the side, those aren't quads in any real sense. They're triples in a pointed graph (or depending on how you like to think about it, a rooted graph or a flow graph).

---
A. Soroka
The University of Virginia Library

On Jul 9, 2015, at 1:21 AM, "James R. Griffin III" <notifications@github.com> wrote:

> In fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java:
> 
> > @@ -1039,6 +1041,32 @@ public void testIngestOnSubtree() throws Exception {
> >  
> >      }
> >  
> > +    /**
> > +     * Ensure that the objects cannot be pairtree child resources
> > +     *
> > +     */
> > +    @Test
> > +    public void testIngestOnPairtree() throws Exception {
> > +        HttpPost method = postObjMethod("");
> > +        HttpResponse response = client.execute(method);
> > +
> > +        //  Following the approach undertaken for FedoraExportIT#shouldRoundTripOnePairtree
> > +        final String objName = response.getFirstHeader("Location").getValue();
> > +        final String pairtreeName = objName.substring(serverAddress.length(), objName.lastIndexOf('/'));
> 
> @ajs6f I'm uncertain as to whether this is the case, but I am currently struggling to find this. Please see the following Gist for a listing of all quads found in respect to the resource "within" the pair-tree "container": https://gist.github.com/jrgriffiniii/598643e2c96379cfaf15
> 
> �
> Reply to this email directly or view it on GitHub.
>